Vagrantfile

```diff
@@ -1,11 +1,82 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-Vagrant.configure("2") do |config|
-  config.vm.box = "precise64"
-  config.vm.box_url = "http://files.vagrantup.com/precise64.box"
-  config.vm.provider :virtualbox do |vb|
-    vb.customize ["modifyvm", :id, "--memory", 2048, "--cpus", 2]
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "ubuntu/trusty64"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+  # config.vm.network "forwarded_port", guest: 80, host: 8080
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  # Create a public network, which generally matched to bridged network.
+  # Bridged networks make the machine appear as another physical device on
+  # your network.
+  # config.vm.network "public_network"
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  # config.vm.synced_folder "../data", "/vagrant_data"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+    # Customize the amount of memory on the VM:
+    vb.memory = "2048"
   end
-  config.vm.provision "shell", path: "vagrant_init.sh"
```

Comment:
Hi Richard,

Unfortunately not in the immediate future. My apologies for the delay, I
have been swamped since returning from PyCon and I have not had time to get
back to this.

On Thu, Jun 16, 2016 at 10:21 AM, Richard Frank <notifications@github.com>
wrote:

> In Vagrantfile
> <https://github.com/quantopian/zipline/pull/1253#discussion_r67386949>:
>
> >    end
> > -  config.vm.provision "shell", path: "vagrant_init.sh"
>
> @Ericlafevers <https://github.com/Ericlafevers> Do you think that's
> something you'll have time to clean up?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/quantopian/zipline/pull/1253/files/be113dff9bb16770d75874ad6f4cb289b99cff92#r67386949>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APOru20N_vLaEekHszMhnxCHsH7V7nIDks5qMYYEgaJpZM4ItJl6>
> .
>